libvast/test/format/arrow.cpp

```diff
@@ -81,9 +81,8 @@ TEST(arrow batch) {
   std::shared_ptr<arrow::RecordBatch> batch;
   while (reader->ReadNext(&batch).ok() && batch != nullptr) {
     REQUIRE_LESS(slice_id, zeek_conn_log.size());
-    auto slice
-      = table_slice{zeek_conn_log[slice_id], table_slice::encoding::arrow,
-                    table_slice::verify::yes};
```

Comment:
How come that the new function doesn't have verification? Not that I want it;  just curious.

Reply:
There's no need to verify again if we move or copy, and if we rebuild we can already rely on the table slice having the correct offsets, otherwise the FlatBufferBuilder would fail in finish.